cpu/stm32f1/cpu.c

```diff
@@ -95,19 +95,25 @@
 #endif
 #endif
 
+#ifndef FW_SLOTS
```

Comment:
Yes, we can consider it as an issue. If the clock is initialised again it doesn't boot.